### PR TITLE
[new release] bam (3 packages) (0.3)

### PR DESCRIPTION
**Pull request by francoisthire**
A property-based testing library with internal shrinking

- Project page: <a href="https://github.com/francoisthire/bam">https://github.com/francoisthire/bam</a>
- Documentation: <a href="https://francoisthire.github.io/bam/">https://francoisthire.github.io/bam/</a>

##### CHANGES:

- Fix a file descriptor leak (francoisthire/bam#8)

- More control over the computation of statistics (francoisthire/bam#8)

- Fix a typing regression with OCaml 5.2 (francoisthire/bam#8)

- Update the dependencies to be compatible with OCaml 5.2 (francoisthire/bam#11)
  (Thanks @mathiasbourgoin, @mattiasdrp)

### CODE REVIEW

File: packages/bam-ppx/bam-ppx.0.3/opam [hunk 1]
```diff
@@ -0,0 +1,42 @@
+opam-version: "2.0"
+synopsis: "A PPX deriving generators for OCaml types"
+description:
+  "Provides a way to automatically get generators for a given type"
+maintainer: ["François Thiré"]
```
Comment (shonfeder) [inline packages/bam-ppx/bam-ppx.0.3/opam:5]: Would you mind adding a contact email here, and in the other packages? E.g.,

```
maintainer: ["François Thiré" <your.email@mail.server>]
```

Comment (francoisthire) [reply packages/bam-ppx/bam-ppx.0.3/opam]: Done.

Comment (shonfeder) [reply packages/bam-ppx/bam-ppx.0.3/opam]: Thank you!

**Comment by shonfeder** [comment]
Thank you for publishing the updates to these packages! Please consider announcing them on discuss.ocaml.org :)

**Pull request merged**